libs/wire-api/src/Wire/API/User/Search.hs

```diff
@@ -200,7 +200,8 @@ data TeamContact = TeamContact
     teamContactSAMLIdp :: Maybe Text,
     teamContactRole :: Maybe Role,
     teamContactScimExternalId :: Maybe Text,
-    teamContactSso :: Maybe ContactSso
+    teamContactSso :: Maybe ContactSso,
+    teamContactEmailUnvalidated :: Maybe Email
```

Comment:
Is this type only visible to admins?  If not, hiding the unvalidated email should be enforced:

```
data TeamContact f = TeamContact
  { ...
    teamContactEmailUnvalidated :: f Email
  }

...
userSearch :: ... -> m (TeamContact (Const ()))
...
adminSearch :: ... -> m (TeamContact Id)
...
```